io.github.junalmeida.webapps_manager.json

```diff
@@ -0,0 +1,55 @@
+{
+    "id": "io.github.junalmeida.webapps_manager",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "6.7",
+    "sdk": "org.kde.Sdk",
+    "base": "io.qt.PySide.BaseApp",
+    "base-version": "6.7",
+    "command": "webapps_manager",
+    "finish-args": [
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--device=dri",
+        "--share=network",
+        "--filesystem=host-os",
+        "--filesystem=/var/lib/flatpak",
```

Comment:
why do you need this?

Reply:
To check if flatpak supported browsers are installed.